src/Loop.php

```diff
@@ -223,6 +223,15 @@ public static function supports($feature) {
     }
 
     /**
+     * Get the underlying loop handle.
+     *
+     * @return null|object|resource The loop handle the event loop operates on. Null if there is none.
+     */
+    public static function getLoopHandle() {
```

Comment:
Because FIG made a mistake, do we have too? :-)

If it were one-two bytes at a single place, I wouldn't care. But honestly, _why_ did they mandate same-line braces for everything but methods and classes.

Analyzing from the survey results:
> method_brace_line:
    next: 15
    same: 7
control_brace_line:
    next: 4
    same: 18

Now can we transform that to "consistent_brace_line" (i.e. brace_line style doesn't change):
> consistent: 11 (= 7+4)
> inconsistent: 11 (= 22-11)

Which is pretty much a tie there. So, if they were to mandate consistent bracing, we'd have braces always on the same line.